src/modelfree/score_agent.py

```diff
@@ -12,6 +18,7 @@
 from modelfree.envs.gym_compete import GymCompeteToOurs, game_outcome
 
 score_ex = Experiment('score')
+score_ex_logger = logging.getLogger(__name__)
```

Comment:
Tempting but `__name__` is equal to `__main__` when running interactively which probably isn't what you want. I've just been hardcoding it.